application/src/main/java/org/togetherjava/tjbot/commands/utils/StringDistances.java

```diff
@@ -177,4 +164,17 @@ private static int[][] computeLevenshteinDistanceTable(CharSequence source,
 
         return table;
     }
+
+    private record MatchScore(String candidate, int score) implements Comparable<MatchScore> {
+        @Override
+        public int compareTo(MatchScore otherMatchScore) {
+            int compare = Integer.compare(this.score, otherMatchScore.score);
```

Comment:
`compare` is a bad name. if at all, then `compareResult`. that said, u can call it `byScore` or `byScoreResult` maybe

Reply:
what about scoreDiff?

Reply:
`diff` suggests that it would be the difference, like `foo - bar`. But its not.